apps/financial-aid/web-veita/src/components/Timeline/EmailElement.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { Text, Box, Icon } from '@island.is/island-ui/core'
+import { ApplicationEventType } from '@island.is/financial-aid/shared/lib'
+
+import * as styles from '../History/History.css'
+
+interface Props {
+  email: string
+  event: ApplicationEventType
+}
+
+const EmailElement = ({ email, event }: Props) => {
+  if (
+    event !== ApplicationEventType.REJECTED &&
+    event !== ApplicationEventType.APPROVED
+  ) {
+    return null
+  }
+
+  return (
+    <Box marginBottom={2} className={styles.timelineMessages}>
+      <Icon icon="mail" type="outline" color="blue400" />
+      <Text>{`Tölvupóstur var sendur á netfangið ${email}`}</Text>
```

Comment:
can we state this? Are we sure it was sent, what if it failed?

Reply:
Added a new column to the application_events table to keep track of if an email has been sent for that events.